Modules/PostFilter/ColorBreakdown/ColorBreakdownModule.cs

```diff
@@ -83,7 +85,25 @@ public override bool Setup()
 
 		private void _CreateOutputs()
 		{
-			_outputs = _data.BreakdownItems.Select(x => new ColorBreakdownOutput(x, _data.MixColors)).ToArray();
+			// Initialize the RGB to RGBW converter to null
+			// If this converter is null then the logic below will use normal RGB filtering
+			RGBToRGBWConverter rgbToRGBWConverter = null;
+
+			// If the break down contains four colors then...
+			if (_data.BreakdownItems.Count == 4)

```

Comment:
How does this work if I am using non mixing colors and I have 4 analog colors chosen? How does it know the difference?